package.xml

```diff
@@ -2,15 +2,18 @@
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
   <name>depthai_ctrl</name>
-  <version>0.5.6</version>
+  <version>0.5.7</version>
   <description>DepthAI camera control ROS2 node</description>
   <maintainer email="manuel.segarra-abad@unikie.com">Manuel Segarra-Abad</maintainer>
   <license>TODO: License declaration</license>
 
-  <exec_depend>rclcpp</exec_depend>
-
   <buildtool_depend>ament_cmake</buildtool_depend>
 
+  <depend>rclcpp</depend>
+  <depend>sensor_msgs</depend>
+  <depend>std_srvs</depend>
```

Comment:
Does the depthai ROS package contain a service? If not, then this dependency is not needed.